examples/examples/sloggish/sloggish_subscriber.rs

```diff
@@ -230,7 +230,6 @@ impl Subscriber for SloggishSubscriber {
         let parent = data.and_then(|span| span.parent.as_ref());
         if stack.iter().any(|id| id == span_id) {
             // We are already in this span, do nothing.
-            return;
```

Comment:
Having an empty `if` clause feels _very_ weird to me stylistically. What if we just changed this to
```rust
if !stack.iter().any(|id| id == span_id) {
    let indent = // ...
}
``` instead, and removed the `else`?